tests/Behat/Page/Admin/PaymentMethod/CreatePage.php

```diff
@@ -49,6 +49,18 @@ public function setInterval(string $interval): void
         $this->getDocument()->fillField('Interval', $interval);
     }
 
+    public function loadPaymentMethods(): void
+    {
+        $this->getDocument()->find('css', '#get_methods')->click();
+        $time = 25000;
```

Comment:
How about checking where is the boundary for the value and choose the lower one.

Reply:
I've reduced it and also add check for earlier execution